bin/bump_auto_updater.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+echo "{
+  \"url\": \"https://github.com/headsetapp/headset-electron/releases/download/v$TAG_NO_V/Headset-$TAG_NO_V.zip\"
+}" > auto_updater.json
```

Comment:
where is `$TAG_NO_V` defined? how do we run this file? 
I think that we can force Travis to push a commit back to GitHub using the `after_deploy` step, and the `TRAVIS_TAG` variable.
So this file can also include the stages to configure git on Travis, use the already present `GITHUB_TOKEN` to be login as Daniel Ravina. Edit `auto_updater.json`. Use the [Github API](https://developer.github.com/v3/pulls/#create-a-pull-request) to push to a new branch, create a PR and even merge automatically. Or just push directly to `master` branch. Either way, it can be done automatically.
What would happen if we just update `auto_updater.json` on the same commit that's pushing a new release? Because it'll be easier to just run this script when `npm version patch -m.....` is run.